docs/source/common_issues.rst

```diff
@@ -595,6 +595,40 @@ string literal types or :py:data:`~typing.TYPE_CHECKING`:
 
    results: 'Queue[int]' = Queue()  # OK
 
+But while using :py:data:`~typing.TypeVar`, observe here that it requires parameters
+while using it as a type. And if we implement it in above way it will give error as
+shown in below example.
+
+.. code-block:: python
+
+   from queue import Queue
+   from typing import TYPE_CHECKING, TypeVar
+
+   _T = TypeVar("_T")
+   if TYPE_CHECKING:
+       MyQueueBase = Queue[_T]
+   else:
+       MyQueueBase = Queue
+
+   class MyQueue(MyQueueBase): pass  # error: Missing type parameters for generic type
```

Comment:
As mentioned above, the error only comes when some configuration options are enabled.

Reply:
done

Reply:
done